src/app.js

```diff
@@ -0,0 +1,53 @@
+let list = []
+// const tableBody = document.querySelector("tableBody")
+
+
+const startApp = async () => {
+  const result = await fetchList()
+  if (result) {
+     list = result
+    populateTable()
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/ZeeUnltd/starter/pullRequest?prid=10061291)
```suggestion
    populateTable();
```